src/ray/core_worker/core_worker.cc

```diff
@@ -1030,6 +1030,18 @@ rpc::Address CoreWorker::GetOwnerAddress(const ObjectID &object_id) const {
   return owner_address;
 }
 
+std::vector<rpc::ObjectReference> CoreWorker::GetObjectRefs(
+    const std::vector<ObjectID> &object_ids) const {
+  std::vector<rpc::ObjectReference> refs;
+  for (const auto &object_id : object_ids) {
+    rpc::ObjectReference ref;
+    ref.set_object_id(object_id.Binary());
+    ref.mutable_owner_address()->CopyFrom(GetOwnerAddress(object_id));
+    refs.push_back(ref);
```

Comment:
```suggestion
    refs.push_back(std::move(ref));
```
? Or is it automatically done?